src/backend/gulpfile.js

```diff
@@ -15,84 +15,55 @@ var exports = module.exports = {};
 exports.minifyJs = function(path, cb) {
   const dir = path + '/js/';
 
-  gulp.task('scheduleJs', function() {
-    return gulp.src([dir + 'FileSaver.js', dir + 'social.js', dir + 'scroll.js', dir + 'navbar.js', dir + 'calendar.js', dir + 'popover.js', dir + 'html2canvas.js', dir + 'jquery.lazyload.js', dir + 'icsGen.js'], {allowEmpty: true})
+  function minifyPipeline(dist, src) {
+    return gulp.src(src.map(file => dir + file))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=5138926)